src/exchangeRate.ts

```diff
@@ -0,0 +1,26 @@
+import config from '../config.json'
+
+export interface CurrencyOption {
+  address: string
+  currencyName: string
+  coinCapCurrencyName: string
+}
+
+export const currencies: { [key: string]: CurrencyOption } = config.currencies
+const uri: string = config.coinCapUri
+
+export const fetchExchangeRates = async (): Promise<{
+  [key: string]: number
+}> => {
+  // Create an arry of promises
+  const promisesArr = Object.keys(currencies).map(async currencyCode => {
+    const currencyData = currencies[currencyCode]
+    const response = await fetch(uri + currencyData.coinCapCurrencyName)
+    const { data } = await response.json()
```

Comment:
`Try/catch` here, so even if some of the fetches fails it would be able to return the ones that succeeded.

Reply:
Updated per comment